docs/product-releases.md

```diff
@@ -4,14 +4,14 @@
 
 In the file `app/_data/kong_versions.yml`, every product release and its corresponding versions are listed. Ensure that the `latest` release for each product is marked with `latest: true`. This informs the platform about the release to use for generating evergreen URLs.
 
-Labels provide flexibility in handling releases. Specifying `label: dev` on a release, alters the URL structure from `/<product>/<release number>/**/` to `/<product>/<label>/**/`, such as `/<product>/dev/**/`. Labels are particularly useful for working on unreleased versions of products. Once a label is added to a release, the content can be edited using [conditional rendering](https://docs.konghq.com/contributing/single-sourced-versions/) and merged it into the `main` branch without the need for long-standing release branches. The content will be rendered only in `/<product>/<label>/`.
+Labels provide flexibility in handling releases. Specifying `label: unreleased` on a release, alters the URL structure from `/<product>/<release number>/**/` to `/<product>/<label>/**/`, such as `/<product>/unreleased/**/`. Labels are particularly useful for working on unreleased versions of products. Once a label is added to a release, the content can be edited using [conditional rendering](https://docs.konghq.com/contributing/single-sourced-versions/) and merged it into the `main` branch without the need for long-standing release branches. The content will be rendered only in `/<product>/<label>/`.
```

Comment:
Ignoring the vale error here, it's not accurate - we shouldn't be using relative URLs in internal docs, since these aren't published to prod. 

@Guaris is there a way for certain Vale checks to ignore directories, or is that defined for all checks only? In this case, we'd want Vale to _not_ run the kong.Relativeurls check on anything in the /docs/ directory.